android/app/src/main/java/share/ShareActivity.java

```diff
@@ -0,0 +1,13 @@
+package io.cozy.flagship.mobile.share;
+
+import com.facebook.react.ReactActivity;
+
+
+public class ShareActivity extends ReactActivity {
```

Comment:
If the pluggin allows to move this class, then the convention would be to create it in `android/app/src/main/java/io/cozy/flagship/share` folder

Reply:
- [ ] Updated